journal-temp/modernized/07-chapter-7.adoc

```diff
@@ -948,19 +973,18 @@ and the Lord my God preserved me both from the distemper and from banishment,
 wherein I do humbly confess his power and special providence,
 to his own praise and glory alone.
 
-The second epistle, printed in the year 1665, after the heat of the contagion was over,
-entitled, An epistle for the remnant of Friends and chosen of God,
-whom he has yet preserved to bear their testimony, in and about the city of London.
-To whom this is a testimony of the dear love and tender care which flows forth,
-and is extended toward them,
-and all tender hearts who are concerned in the like sufferings, temptations and trials.
+[.embedded-content-document.epistle]
+--
+
+[.blurb]
```

Comment:
So this part is a mess. haha I took some time to try to figure out what was happening here, and what I came up with is that it's somewhat of an introduction to the epistle maybe? Idk.